sound/soc/intel/boards/Kconfig

```diff
@@ -664,6 +664,7 @@ config SND_SOC_INTEL_SOUNDWIRE_SOF_MACH
 	depends on SOUNDWIRE
 	select SND_SOC_MAX98373_I2C
 	select SND_SOC_MAX98373_SDW
+	select SND_SOC_MAX98363
```

Comment:
nit-pick: try to keep the parts sorted by increasing order, i.e. move this up by two lines.

Reply:
Ack done